opm/simulators/flow/AluGridVanguard.hpp

```diff
@@ -34,10 +34,12 @@
 #include <opm/common/OpmLog/OpmLog.hpp>
 
 #include <opm/grid/CpGrid.hpp>
+#include <opm/grid/cpgrid/LevelCartesianIndexMapper.hh>
```

Comment:
Also related to https://github.com/OPM/opm-grid/pull/766: why did you choose ".hh" as the extension here and not ".hpp"?

Reply:
I wasn't sure about that. For CpGrid, many (maybe all?) headers have the extension ".hpp". For PolyhedralGird, ".hh", and for AluGrid, ".hpp". Then, should I go for the majority and take ".hpp"?

Reply:
Yes, would say that is the correct approach here,  ".hpp" for ../opm/common/LevelCartesianIndexMapper.hpp, for the CpGrid specialization, and for ALuGrid, and only ".hh" for PolyhedralGrid specialization.